app/models/auth/AuthState.scala

```diff
@@ -12,7 +12,12 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *@
+ */
 
-@()(implicit lang: Lang, request: Request[_], templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
-<link rel='stylesheet' href='@routes.Assets.at("tamc.css")'/>
+package models.auth
+
+sealed abstract class AuthState(val isLoggedIn: Boolean, val permanent: Boolean)
```

Comment:
I made it sealed abstract so that in future changes people are less likely to attempt to create instances of this if they make changes in the future. I can see your point however, and will consider changing this